### PR TITLE
Added search center URL property to xSPSearchServiceApp

### PR DESCRIPTION
**Pull request by BrianFarnhill**

### CODE REVIEW

**Comment by BrianFarnhill** [comment]
@caadam @camiloborges @anlynes @ykuijs Code review time - smaller one here, just added a search center URL property to the existing xSPSearchServiceApp resource

File: Tests/xSharePoint/xSharePoint.xSPSearchServiceApp.Tests.ps1 [hunk 1]
```diff
@@ -251,5 +251,83 @@ Describe "xSPSearchServiceApp" {
...
+            Mock New-Object {
+                return @{
+                    DefaultGatheringAccount = "DOMAIN\username"
+                }
+            } -ParameterFilter { $TypeName -eq "Microsoft.Office.Server.Search.Administration.Content" }
+            
+            It "should return false from the test method" {
+                Test-TargetResource @testParams | Should Be $false
+            }
+        }
```
Comment (ykuijs) [inline Tests/xSharePoint/xSharePoint.xSPSearchServiceApp.Tests.ps1]: No test for the Set-TargetResource method?

**Comment by ykuijs** [comment]
One comment, the rest looks good

File: Tests/xSharePoint/xSharePoint.xSPSearchServiceApp.Tests.ps1 (hunk 1 above)
Comment (BrianFarnhill) [inline Tests/xSharePoint/xSharePoint.xSPSearchServiceApp.Tests.ps1]: Thanks @ykuijs - Just pushed this one in now. I'll merge and close after the tests light up.

**Pull request merged**